### PR TITLE
silence plugin server sentry logspam

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

The plugin server is spamming Sentry a lot with race condition stuff that's expected. This PR removes a bit more than I'd like but we need a more elegant solution in the future to filter what we send to Sentry or not.

## How did you test this code?

*Please describe.*

### CODE REVIEW

**Comment by neilkakkar** [comment]
Quick q: Do we have metrics that capture how often these errors happen?

File: plugin-server/src/utils/metrics.ts [hunk 1]
```diff
@@ -14,7 +14,6 @@ export async function instrumentQuery<T>(
     try {
         return await runQuery()
     } catch (error) {
-        Sentry.captureException(error, { extra: { query_tag: tag } })
```
Comment (yakkomajuri) [inline plugin-server/src/utils/metrics.ts:17]: this alone will go a long way

**Comment by yakkomajuri** [comment]
> Quick q: Do we have metrics that capture how often these errors happen?


Some do, some don't.

File: plugin-server/src/utils/metrics.ts (hunk 1 above)
Comment (neilkakkar) [reply plugin-server/src/utils/metrics.ts:17]: can remove the catch block completely?

Comment (yakkomajuri) [reply plugin-server/src/utils/metrics.ts:17]: indeed we can

**Pull request merged**